src/types/GovernanceTypes.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.30;
+
+/*///////////////////////////////////////////////////////////////
+                        Structs
+//////////////////////////////////////////////////////////////*/
+
+/// @notice Struct to store rollback data.
+/// @param queueExpiresAt The timestamp before which the rollback must be queued for execution.
+/// @param executableAt The timestamp after which the rollback can be executed.
+/// @param executed Whether the rollback has been executed.
+/// @param canceled Whether the rollback has been canceled.
+/// @dev executed and canceled are mutually exclusive - both cannot be true
+///      queueExpiresAt must be > block.timestamp when rollback is proposed
+///      executableAt must be > block.timestamp when rollback is queued
+struct Rollback {
+  uint256 queueExpiresAt;
+  uint256 executableAt;
```

Comment:
The Governor calls this field `etaSeconds`. I'm debating in my head if we should match [that name](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/Governor.sol#L44). I like yours better, but is it worth being consistent with the names here too 🤔

Reply:
Also created an issue to bikeshed this more later. But good to resolve this thread 👍